src/main/configSchema.ts

```diff
@@ -0,0 +1,140 @@
+export type ConfigurationSchema = {
+    storagePath: string,
+    bufferStoragePath?: string,
+    retailLogPath?: string,
+    classicLogPath?: string,
+    maxStorage: number,
+    monitorIndex: number,
+    selectedCategory: number,
+    audioInputDevice?: string,
+    audioOutputDevice?: string,
+    minEncounterDuration: number,
+    startUp?: boolean,
```

Comment:
I think this doesn't need to be optional

Reply:
They were copy/pasted from `configService.ts`, so I didn't change them, but I can surely make them non-optional anyway :)